src/vstart.sh

```diff
@@ -754,10 +764,15 @@ start_osd() {
     for osd in `seq 0 $(($CEPH_NUM_OSD-1))`
     do
 	if [ "$new" -eq 1 ]; then
-		    wconf <<EOF
+                    wconf <<EOF
```

Comment:
if you want to take this opportunity to fix the awful indent in this script, i'd suggest use 2 or 4 spaces, but not 20/4.

Reply:
I'll  try to do it. I referred to some other projects' scripts, both 2 and 4 spaces exist in the same script. But I wonder if it would be better to use just 4 spaces for vstart.sh?

Reply:
either way, we use both 2 and 4 in this very script =(